### PR TITLE
Add `patientId` and `facilityId` columns in `CallResult` table 

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.shortcut.com/simpledotorg/story/9058/add-sync-support-for-pulling-overdue-call-results

### CODE REVIEW

**Comment by janhavisinghh** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4218** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/4218" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/4218?utm_source=stack-comment).

File: app/src/main/java/org/simple/clinic/overdue/callresult/CallResult.kt [hunk 1]
```diff
@@ -54,7 +58,9 @@ data class CallResult(
         removeReason = null,
         outcome = Outcome.AgreedToVisit,
         timestamps = Timestamps.create(clock),
-        syncStatus = syncStatus
+        syncStatus = syncStatus,
+        patientId = appointment.patientUuid,
+        facilityId = appointment.facilityUuid
```
Comment (msasikanth) [inline app/src/main/java/org/simple/clinic/overdue/callresult/CallResult.kt:63]: The appointment facility id is not always the same as the user's current facility. Did we check Prabhanshu to see what we want here?

File: app/src/main/java/org/simple/clinic/overdue/callresult/CallResult.kt [hunk 2]
```diff
@@ -87,7 +95,9 @@ data class CallResult(
         removeReason = removeReason,
         outcome = Outcome.RemovedFromOverdueList,
         timestamps = Timestamps.create(clock),
-        syncStatus = syncStatus
+        syncStatus = syncStatus,
+        patientId = appointment.patientUuid,
+        facilityId = appointment.facilityUuid
```
Comment (msasikanth) [inline app/src/main/java/org/simple/clinic/overdue/callresult/CallResult.kt:100]: <i class='graphite__hidden'>[Re: lines 79 to 100]<br/></i>

Same comment as above<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/4218?utm_source=unchanged-line-comment">Graphite</a>.</span>

File: app/src/androidTest/java/org/simple/clinic/storage/migrations/Migration105AndroidTest.kt [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import org.junit.Test
+import org.simple.clinic.assertColumns
+
+class Migration105AndroidTest : BaseDatabaseMigrationTest(
+    fromVersion = 104,
+    toVersion = 105
+) {
+
+  @Test
+  fun patient_id_and_facility_id_columns_should_be_added_to_call_result_table() {
```
Comment (msasikanth) [inline app/src/androidTest/java/org/simple/clinic/storage/migrations/Migration105AndroidTest.kt:12]: Looks like you missed removing facility id from the test name

**Review by msasikanth — CHANGES_REQUESTED** [review]
See inline comments

**Review by msasikanth — APPROVED** [review]

**Review by msasikanth — APPROVED** [review]

**Pull request merged**